crates/ironrdp-connector/src/connection.rs

```diff
@@ -625,7 +625,7 @@ fn create_gcc_blocks<'a>(
 
     ClientGccBlocks {
         core: ClientCoreData {
-            version: RdpVersion::V5_PLUS,
+            version: RdpVersion::V10_12,
```

Comment:
Hmm, can't this change break something down the line? Is this required for display control functionality?

Reply:
That was leftover from me trying something out, reverted in 4d878fbcf6608dcb19161ea62e2f36cffd8a28eb